ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/snapshot/services/MemberReduceOperator.java

```diff
@@ -0,0 +1,16 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.snapshot.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.entities.Member;
+
+import java.util.function.BinaryOperator;
+
+public class MemberReduceOperator implements BinaryOperator<Member> {
+	@Override
+	public Member apply(Member member, Member member2) {
+
+		if (member.getTimestamp().isAfter(member2.getTimestamp()))
```

Comment:
just to double check.. timestamp is a mandatory value on the member right?

Reply:
well, actually not. timestamppath is an optional attribute according to the ldes spec. I'll add validation of the members and exception handling in a follow-up PR on branch `feature/VSDSPUB-564-snapshot-exceptions`